Customer.AppServices/Validator/UpdateCustomerDtoValidator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Customer.AppModels.Dtos;
+using FluentValidation;
+
+namespace Customer.AppServices.Validator
+{
+    public class UpdateCustomerDtoValidator : AbstractValidator<UpdateCustomerDto>
+    {
+        public UpdateCustomerDtoValidator()
+        {
+            RuleFor(x => x.FullName).NotEmpty().WithMessage("Currency required");
+
+            RuleFor(x => x.Email)
+                    .NotEmpty().WithMessage("Currency Required")
+                    .EmailAddress().WithMessage("Invalid Email");
+
+            RuleFor(x => x.Cpf)
+                    .NotEmpty().WithMessage("Currency required")
+                    .Must(cpf => CpfValidator.IsCpfValid(cpf)).WithMessage("CPF is invalid");
+
+            RuleFor(x => x.Cellphone).NotEmpty().WithMessage("Currency required");
+
+            RuleFor(x => x.DateOfBirth)
+                    .NotEmpty()
+                    .GreaterThan(DateTime.MinValue)
+                    .WithMessage("Currency required");
```

Comment:
1. O que é Currency?
2. Métodos próprios do FluentValidation já provem excelentes mensagens, não precisamos sobrescreve-las 
```suggestion
            RuleFor(x => x.DateOfBirth)
                    .NotEmpty()
                    .GreaterThan(DateTime.MinValue);
```